dev.Dockerfile

```diff
@@ -13,10 +13,12 @@
 #   # you can run the tests against this new image using:
 #   ./test -i kaggle/python-dev -p test_user_secrets.py
 #
-FROM gcr.io/kaggle-images/python:staging
 
+#FROM gcr.io/kaggle-images/python:staging
```

Comment:
Did you mean to comment this line?

Reply:
Sorry, it is a merge issue. Changed back.